content/client-lib-development-guide/features.textile

```diff
@@ -558,6 +558,7 @@ h3(#realtime-presence). Presence
 * @(RTP11)@ @Presence#get@ function:
 ** @(RTP11a)@ Returns the list of current members on the channel in a callback. By default, will wait for the @SYNC@ to be completed, see "RTP11c1":#RTP11c1
 ** @(RTP11b)@ Implicitly attaches the @Channel@ if the channel is in the @INITIALIZED@ state. However, if the channel is in or enters the @DETACHED@ or @FAILED@ state before the operation succeeds, it will result in an error
+** @(RTP11d)@ If the @Channel@ is in the @SUSPENDED@ state then the @get@ function immediately returns the members stored in the @PresenceMap@ and does not wait until the channel state changes to @ATTACHED@ irrespective of whether @waitForSync@ is true or false
```

Comment:
I'm not sure I agree with this. When you're `SUSPENDED` then there is no synchronisation of state between the client and server. I don't see why it's valid to continue to return stale presence state.